client/extensions/woocommerce/app/dashboard/store-move-notice-view.js

```diff
@@ -21,39 +21,25 @@ import config from '@automattic/calypso-config';
 
 import megaphoneImage from 'calypso/assets/images/woocommerce/megaphone.svg';
 
-function getStoreStatus( isStoreDeprecated, isStoreRemoved ) {
-	if ( isStoreDeprecated ) {
-		return 'store-deprecated';
-	}
-
-	if ( isStoreRemoved ) {
-		return 'store-removed';
-	}
```

Comment:
I am leaning towards keeping this so that it serves as a reminder when we'd want to change to `store-removed` later on. Either that or maybe create a separate task to remind so that we can guarantee it's not missed.

Reply:
That's a good point 👍 Reverted the change in https://github.com/Automattic/wp-calypso/commit/725f582efe0c3e5c2cb6cdd7171a9652dd064659